test/Microsoft.ML.TestFramework/Attributes/TensorflowFactAttribute.cs

```diff
@@ -12,14 +13,17 @@ namespace Microsoft.ML.TestFramework.Attributes
     /// </summary>
     public sealed class TensorFlowFactAttribute : EnvironmentSpecificFactAttribute
     {
-        public TensorFlowFactAttribute() : base("TensorFlow is 64-bit only")
+        public TensorFlowFactAttribute() : base("TensorFlow is 64-bit only and is not supported on Linux with libc < v2.23")
         {
         }
 
         /// <inheritdoc />
         protected override bool IsEnvironmentSupported()
         {
-            return Environment.Is64BitProcess;
+            return Environment.Is64BitProcess &&
+                   ( RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ||
+                     AttributeHelpers.CheckLibcVersionGreaterThanMinimum(new Version(2, 23)));
```

Comment:
I know this is the libc version that is the minimum Onnx supported version, but is it the same for TensorFlow?

Reply:
Yeah, it's also the same case for TF.

Reply:
Strictly speaking, not exactly the same, but whatever applies to disable Onnx fact also applys to tf fact.